R/get_coverage_wcs.R

```diff
@@ -4,7 +4,7 @@
 #' from which it is read with `terra::rast()` - if needed reprojected -
 #' and returned as a `SpatRaster`object
```

Comment:
```suggestion
#' and returned as a `SpatRaster` object
```

(This is actually not part of this PR)